pkgs/development/misc/vc4/binutils.nix

```diff
@@ -0,0 +1,13 @@
+{ stdenv, texinfo, vc4-gcc-stage1, vc4-binutils, flex, bison, fetchFromGitHub }:
```

Comment:
We no longer need these files, right?

Reply:
i need to try deleting things and see what breaks or keeps working

Reply:
the seperate binutils and gcc have now been removed, and newlib updated to use the patched ones